plugins/modules/dconf.py

```diff
@@ -401,11 +405,23 @@ def main():
         argument_spec=dict(
             state=dict(default='present', choices=['present', 'absent', 'read']),
             key=dict(required=True, type='str', no_log=False),
-            value=dict(required=False, default=None, type='str'),
+            # Converted to str below after special handling of bool.
+            value=dict(required=False, default=None, type='raw'),
```

Comment:
The documentation must also say `type: raw`. Add some explanation that the values are converted to strings.

Reply:
I think it's better to use `type: raw`. The documentation shouldn't be that complex, I'll happily add some suggestions (but I cannot right now because there's no change close enough to it).